e2e/pages/createCase/createCase.js

```diff
@@ -6,13 +6,24 @@ module.exports = {
     jurisdiction: 'jurisdiction',
     caseType: 'case-type',
     event: 'event',
+    enterCaseName: {
+      caseName: '#caseName',
+    },
   },
+  testCaseName: 'Barnet Council v Smith',
+  continueButton: 'Continue',  
   startButton: 'Start',
 
+  enterCaseName() {
+    I.fillField(this.fields.enterCaseName.caseName, this.testCaseName);
```

Comment:
i dont know if the structure here should be like this, so do you need `enterCaseName` inside fields? could you just have `Fields > caseName`?